qt/widgets/common/src/QtPropertyBrowser/ParameterPropertyManager.cpp

```diff
@@ -77,9 +77,9 @@ void ParameterPropertyManager::setDescription(QtProperty *property,
  * @param property :: Property to clear error for
  */
 void ParameterPropertyManager::clearError(QtProperty *property) {
-  m_errors.remove(property);
-  emit propertyChanged(property);
-  updateTooltip(property);
+		m_errors.remove(property);
```

Comment:
A test for `hasProperty(property)` is missing.

Reply:
ah yes. Thank you